src/cljs/coffeepot/app.cljs

```diff
@@ -17,7 +22,36 @@
 
 (firebase/add-auth-listener)
 
-(defn main-panel []
+(defn hook-browser-navigation! []
+  (doto (History.)
+    (goevents/listen
+     EventType/NAVIGATE
+     (fn [event]
+       (secretary/dispatch! (.-token event))))
+    (.setEnabled true)))
+
+(defn app-routes []
+  (secretary/set-config! :prefix "#")
+
+  (defroute "/" []
+    (re-frame/dispatch [::events/set-page :front]))
+
+  (defroute "/coffeepot" []
+    (re-frame/dispatch [::events/set-page :coffeepot]))
+
+  (hook-browser-navigation!))
+
+(defmulti current-page #(let [page (re-frame/subscribe [::subs/active-page])]
+                          (if (nil? @page) :default @page)))
+(defmethod current-page :front []
+  [front/front-page])
+(defmethod current-page :coffeepot []
+  [ui/mui-theme-provider {:mui-theme theme/coffeepot-theme} [coffee/app-main]])
+(defmethod current-page :default []
+  [ui/mui-theme-provider {:mui-theme theme/coffeepot-theme} [front/login-page]]
```

Comment:
Hey. Why is this row still here? Missing commit?

Reply:
Don't know. Be gone you foolish row.